test/test_client.py

```diff
@@ -1,7 +1,9 @@
+from __future__ import division, print_function
 import types
 import unittest
+import time
 import datetime
-import urlparse
+from six.moves.urllib import parse as urlparse
```

Comment:
I feel like `from six.moves import urllib` then using `urllib.parse` might make things easier to grep for in the future, would you be OK changing to this here and in the other files?